contributed_definitions/NXapm.nxdl.xml

```diff
@@ -447,41 +474,72 @@
                    microscopy) are accelerated towards a position-sensitive  
                    and time-of-flight taking detector system.  
                    Hence, there is no real probe/beam.  
-                 * Second, the specimen is the lens of an atom probe microscope.  
+                 * Second, the specimen is the lens of the microscope.  
             </doc>
             <field name="instrument_name" type="NX_CHAR">
                 <doc>
-                     Given name of the atom probe at the hosting institution. This is an 
+                     Given name of the atom probe at the hosting institution. This is an
                      alias. Examples could be LEAP5000, Raptor, Oxcart, one atom at a time,
                      etc.
                 </doc>
             </field>
-            <field name="location" optional="true" type="NX_CHAR">
+            <field name="location" type="NX_CHAR" optional="true">
                 <doc>
                      Location of the lab or place where the instrument is installed.
                      Using GEOREF is preferred.
                 </doc>
             </field>
-            <group type="NXmanufacturer"/>
+            <group type="NXmanufacturer">
+                <field name="name" type="NX_CHAR"/>
+                <field name="model" type="NX_CHAR"/>
```

Comment:
should not these be required already in the base class, so we shall not mention them in every appdef?

Reply:
yes, and also recommended can be added to BASE class, so nothing needs to be repeated in the appdef